src/api/test/unit/package_remove_test.rb

```diff
@@ -0,0 +1,119 @@
+# Testing the things that should and shouldn't happen when you remove a Package
+require File.expand_path(File.dirname(__FILE__) + "/..") + "/test_helper"
+
+class PackageRemoveTest < ActiveSupport::TestCase
+
+  fixtures :all
+
+  def setup
```

Comment:
This is also set by [Suse::Backend.start_test_backend](https://github.com/openSUSE/open-build-service/blob/master/src/api/lib/opensuse/backend.rb#L204)